client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-edit/components/committee-detail-edit/committee-detail-edit.component.ts

```diff
@@ -89,7 +89,11 @@ export class CommitteeDetailEditComponent extends BaseComponent implements OnIni
      * @returns A function that will return a boolean
      */
     public getDisableOptionWhenFn(): (value: Selectable) => boolean {
-        return value => value.id === this.committeeId;
+        return value => value.id === this.committeeId || !(this.isOrgaManager || this.committeeRepo.getViewModel(value.id).manager_ids?.includes(this.operator.user.id));
+    }
+
+    public getDisableOptionWhenNotComAdmin(): (value: Selectable) => boolean {
+        return value => !(this.isOrgaManager || this.committeeRepo.getViewModel(value.id).manager_ids?.includes(this.operator.user.id));
```

Comment:
Same thing here, this doesn't check for higher level committee admin rights.
Also this code looks almost exactly like the one above. Perhaps use this as a helper function in `getDisableOptionWhenFn`?